hw10_functions.py

```diff
@@ -0,0 +1,53 @@
+# 1.Перепишіть наш калькулятор (який ми робили  в іншій домашці) за допомогою функцій.
+#
+
+number_1 = int(input('Enter first number :'))
+action = input('Enter action (you can use: +,-,/,* ) ')
+number_2 = int(input('Enter second number : '))
+
+def add_number(number_1, number_2):
+    print(number_1 + number_2)
+
+def substract_number(number_1, number_2):
+    print(number_1 - number_2)
+
+def multiply_number(number_1, number_2):
+    print(number_1 * number_2)
+
+def divide_number(number_1, number_2):
+    if number_2 == 0:
+        print('Division by zero')
+    else:
+        print(number_1 / number_2)
+
+if action == "+" or action == "-" or action == "/" or action == "*":
+    if action == "+":
+        add_number(number_1, number_2)
+    elif action == "-":
+        substract_number(number_1, number_2)
+    elif action == "*":
+        multiply_number(number_1, number_2)
+    elif action == "/":
+        divide_number(number_1, number_2)
+else:
+    print("Incorrect action")
```

Comment:
Ok, but can be better. With our knowledge I want to see something like this
https://github.com/MarinaZelenska/test_repo_python/pull/7/files#diff-e38849d79e2ad0d5aea391b4c687352c5b6db1eea0f7c0e6db2286b3078b2314